config/configmapprovider/testdata/expand-with-all-env.yaml

```diff
@@ -1,5 +1,12 @@
 test_map:
   extra: "$EXTRA"
+  env:
+    recv.1: "$ENV_VALUE_1"
+    recv.2: "${ENV_VALUE_2:default value for 2 not substituted}"
```

Comment:
[Bash](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion) and [zsh](https://zsh.sourceforge.io/Doc/Release/Expansion.html#Parameter-Expansion) use the `${parameter:-word}` syntax (${name-word}` works too on zsh). I would prefer to stick to the known convention for this.

Reply:
Noted. I have changed the syntax to `${parameter:-word}`